plugin.yaml

```diff
@@ -196,7 +201,8 @@ dsl_definitions:
       default: { get_property: [ SELF, tags ] }
     auto_tags:
       default: { get_property: [ SELF, auto_tags ] }
-
+    number_of_attempts:
+       default: { get_property: [ SELF, retry_number ] }
```

Comment:
Since we are already touching this list, do you want to add the `store_facts` that I left out?

Reply:
Sure